osfoffline/polling_osf_manager/osf_query.py

```diff
@@ -19,8 +19,8 @@
 class OSFQuery(object):
     def __init__(self, loop, oauth_token):
         self.headers = {
-            # 'Authorization': 'Bearer {}'.format(oauth_token),
-            'Cookie': 'osf_staging={}'.format(oauth_token)
+            'Authorization': 'Bearer {}'.format(oauth_token),
+            # 'Cookie':'osf_staging={}'.format(oauth_token)
```

Comment:
Remove the line referencing cookie. If we want it back we can look at the git history.